README.md

```diff
@@ -148,7 +149,7 @@ When troubleshooting, you can run hiera from the commandline with the `--explain
 
 ## How it's secure by default
 
-In order to prevent accidental leakage of your secrets throughout all of the locations puppet stores information the returned value of the `azure_key_vault::secret` function & Hiera backend return a string wrapped in a Sensitive data type.  Lets look at an example of what this means and why it's important.  Below is an example of pulling a secret and trying to output the value in a notice function.
+In order to prevent accidental leakage of your secrets throughout all of the locations puppet stores information the returned value of the `azure_key_vault::secret` function returns a string wrapped in a Sensitive data type.  Lets look at an example of what this means and why it's important.  Below is an example of pulling a secret and trying to output the value in a notice function.
```

Comment:
Hey, 

Thanks for pointing this out!  Since this module is supposed to be safe by default I have pushed a new release that now returns hiera lookups wrapped in a sensitive data type.  

Since this wasn't being done before i have also bumped the major version of the module since this is a breaking change.

https://github.com/TraGicCode/tragiccode-azure_key_vault/pull/96

Reply:
I undid this change.